x/evm/types/params.go

```diff
@@ -49,6 +49,9 @@ var (
 		"channel-31", // Cronos
 		"channel-83", // Kava
 	}
+	DefaultCreateWhitelistAddresses = []string{
+		"0xC6Fe5D33615a1C52c08018c47E8Bc53646A0E101",
```

Comment:
what's this? The param defaults should be empty

Reply:
that was just for testing - will remove